crates/http/Cargo.toml

```diff
@@ -8,9 +8,9 @@ authors = ["DeisLabs Engineering Team"]
 doctest = false
 
 [dependencies]
-wasmtime = "0.35"
+wasmtime = "0.39.1"
 anyhow = "1"
-wit-bindgen-wasmtime = { git = "https://github.com/Mossaka/wit-bindgen", rev = "8252b0e39c7495f647ec0b0898721a7c641fc6c8" }
+wit-bindgen-wasmtime = { git = "https://github.com/Mossaka/wit-bindgen", rev = "f7734c4c5c306bd8f0c047bb4eba782c5cdbc416" }
```

Comment:
can we use a tag here too?

Reply:
I would rather keep this rev here since this is from my forked repo. This allows me to track which exact commit are we using in slight.

Reply:
Why are we using your forked repo? Will you be contributing your work upstream?

Reply:
You can see what I added to wit-bindgen. https://github.com/bytecodealliance/wit-bindgen/compare/main...Mossaka:wit-bindgen:pub_fields

Probably not because this is not considered a good practice to make fields public. I'd convert this to an issue and figure out another way to solve this.